src/Database/LSMTree/Internal/PageAcc.hs

```diff
@@ -125,7 +125,7 @@ pageSize = 4096
 -- Checking entry size allows us to use 'Word16' arithmetic, we don't need to
 -- worry about overflows.
 --
-sizeofEntry :: SerialisedKey -> Entry SerialisedValue BlobSpan -> Int
+sizeofEntry :: SerialisedKey -> Entry SerialisedValue blob -> Int
```

Comment:
Hmm, we want to know how large this entry is going to be on the page, where according to page format spec it will always use 12 bytes for the blob span.
I'm just changing this because I needed to calculate the size of an entry of `SerialisedBlob`. Once I add it to the `RunBuilder`, it will become a 12 byte blob span. I don't have that `BlobSpan` available at this point, but I should still be able to know how large the entry is going to be on the page.

Reply:
Ah, true. It used to be on `SerialisedBlob`, but I changed where we read the blob. So I can just map the selector `BlobRef -> BlobSpan`.

I still think the page format size calculation doesn't really depend on the type used to represent the blob in memory, but if the generality isn't needed, keeping the fixed `BlobSpan` could avoid potential confusion. :+1: